api/organisations/tests/test_models.py

```diff
@@ -63,11 +61,14 @@ def test_cancel_subscription_cancels_chargebee_subscription(
     ):
         # Given
         organisation = Organisation.objects.create(name="Test org")
-        subscription = Subscription.objects.create(
-            organisation=organisation,
-            payment_method=CHARGEBEE,
-            subscription_id="subscription-id",
-        )
+
+        subscription = Subscription.objects.get(organisation=organisation)
```

Comment:
Minor: you could probably neaten this up slightly by doing: 

```
Subscription.objects.filter(organisation=organisation).update(...)
```

Reply:
Yeah, done